pytest_itde/__init__.py

```diff
@@ -61,6 +61,17 @@
     ),
 )
 
+SSH = config.OptionGroup(
+    prefix="ssh",
+    options=(
+        {
+            "name": "port",
+            "type": int,
+            "default": Ports.forward.ssh,
```

Comment:
not related to the change, but we maybe need to think about what the plugin should do. FYI @Nicoretti 

I think, in the normal case the test should simply get a session wide database with random ports and should be able to start also new databases witb random ports, because using default ports in tests is in general somewhat dangerous.
In that case, we only need port cli parameter, in case we want to supply a database from the outside,

Reply:
I created ticket #357 
Please review / fix grammar of last statement.